daniel/pass/ps.c

```diff
@@ -0,0 +1,83 @@
+#include "ps.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <dirent.h>
+
+int_array getProcesses() {
+    DIR *directory;
+    int_array pidList;
+    int counter = 0;
+    struct dirent *file;
+    directory = opendir("/proc");
+    if (directory) {
+        while ((file = readdir(directory)) != NULL) {
+            if (atoi(file->d_name)) {
+                pidList.data[counter++] = atoi(file->d_name);
```

Comment:
Again, inline stuff makes your code spaghetti.  Separate it.

Reply:
Still don't like the inline stuff, it's not worth it for the readability of your code.